app/models/feedback_message.rb

```diff
@@ -1,8 +1,7 @@
 class FeedbackMessage < ApplicationRecord
-  belongs_to :offender, foreign_key: "offender_id", class_name: "User", optional: true, inverse_of: :feedback_messages
-  belongs_to :reviewer, foreign_key: "reviewer_id", class_name: "User", optional: true, inverse_of: :feedback_messages
```

Comment:
Removed `reviewer` association because there is no `reviewer_id` in the `feedback_messages` table, so the association seems to be unused.